moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -2,6 +2,13 @@ import React, { useEffect } from "react";
 import { v4 as uuidv4 } from "uuid";
 import { TextField } from "@material-ui/core";
 
+
+/*
+ * Socrata Endpoint
+ */
+export const SOCRATA_ENDPOINT =
+  "https://data.austintexas.gov/resource/p53x-x73x.geojson?$select=signal_id,location_name,location,signal_type,id&$order=signal_id asc&$limit=9999";
+
```

Comment:
Probably irrelevant, but is it safe to assume both production and staging need to pull data from here?

Reply:
yes—this is the canonical source regardless of environment.